.gitignore

```diff
@@ -9,3 +9,4 @@ dist/
 firebase-debug.log
 node_modules/
 package-lock.json
+.DS_Store
```

Comment:
This is not really specific to this repository, but rather something you’d want to ignore globally (in `~/.gitignore`, similar to `Desktop.ini`, `Thumbs.db`, etc.).

Reply:
If you prefer to add this, then let’s add other common ones too, e.g.

```
# Folder view configuration files
.DS_Store
Desktop.ini

# Thumbnail cache files
._*
Thumbs.db
```

Reply:
ah yeah sg 👍